items.json

```diff
@@ -22171,9 +22171,9 @@
     "name": "Reaper",
     "culture": "Neutral",

```

Comment:
I think you'll probably want to remove onehanded mode from some of these weapons

Reply:
removed 1h modes from most weps